### ISSUE TITLE
Add possibility for embedded images

### ISSUE THREAD

**Issue by SupaBasti** [cjwirth/RichEditorView#106]
I extended rich_editor.js by a function to embed images:

RE.embeddImage = function(imagedata, alt) {
    var img = document.createElement('img');
    img.setAttribute("src", "data:image/png;base64," + imagedata);
    img.setAttribute("alt", alt);
    img.setAttribute("docID", "")
    img.onload = RE.updateHeight;
    
    RE.insertHTML(img.outerHTML);
    RE.insertHTML('<br>');
    RE.callback("input");
};

I need flexibility to add certain attributes, e.g. the docID from the example

General question about images: how can I succeed in scrolling to bottom after image got inserted? It works for text, but images seem to get scrolled only half way down.

**Comment by cjwirth** [comment]
No, I'm not going to add your special attributes into an official release. 

However -- it is possible to add your own scripts to the RichEditorView, so then you can include your own JS functions and do whatever you need!

> how can I succeed in scrolling to bottom after image got inserted? It works for text, but images seem to get scrolled only half way down.

:confused: I don't really know, I've never encountered this, sorry!
